pages/understanding-json-schema/reference/type.md

```diff
@@ -129,7 +129,7 @@ The `type` keyword can take two forms:
 
 1. **A single string**. When it is a single string, it must be one of the types mentioned above (`array`, `boolean`, `integer`, `number`, `null`, `object`, `regular expressions`, or `string`). This specifies that the instance data is only valid when it matches that specific type. 
 
-Here is an example of using the `string` keyword as a single string:
+Here is an example of using the `number` keyword as a single string:
```

Comment:
We usually only refer to the property names as "keywords", not their values. So, `type` is a keyword and `"number"` is the value of that keyword.

```suggestion
Here is an example of using `"number"` as a single string value:
```